torchbox/torchbox.com
# Add settings for the cookie page and the digital emissions page

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1471999300)

### Description of Changes Made

- Instead of hard-coding links, move them to a settings page where we can select a page. Then if the page is moved for any reason, the link will still work.
- We'll need to update the links straight away after deployment.

### How to Test

- Visit http://localhost:8000/admin/settings/torchbox/importantpagesettings/1/ and select links for 'cookie policy page' and 'carbon emissions page'.

- In the footer check that the 'Find out more about our carbon impact' link points to the page you chose for the carbon emissions page.

- In an incognito window, load any page and check that the 'Find out more' link in the cookie banner points to the page you chose for 'cookie policy page'.

- Try removing the links in the settings, and check that the relevant text and links are completely removed.

### Screenshots

<details>
  <summary>Expand to see more</summary>

Carbon emissions link:

![Screenshot 2024-07-22 at 14 51 05](https://github.com/user-attachments/assets/a605af29-ef0c-47e1-84d2-fc8c4e18b8c1)

Cookie policy link:

![Screenshot 2024-07-22 at 14 51 39](https://github.com/user-attachments/assets/4ed50e41-57f4-4a6a-aa39-c52d71f4c9f2)



</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
